frontend/components/molecules/Form.js

```diff
@@ -0,0 +1,112 @@
+import getImage from '../../utils/getImage'
+import Link from 'next/link'
+import * as timeago from 'timeago.js';
+import Tag from '../atoms/Tag'
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@reach/tabs";
+import Button from '../atoms/Button';
+import { useRouter } from 'next/router';
+
+export default function Form({
+  id,
+  author,
+  author_id,
+  is_active,
+  icon,
+  banner,
+  title,
+  description,
+  publish_time,
+  accessibility,
+  limitations,
+  answer_count
+}) {
+  const router = useRouter()
+  const bannerImg = getImage({ id, banner, type: 'banner' })
+  const iconImg = getImage({ id, icon, type: 'icon' })
+
+  return (
+    <div>
+      <div
+        className="bg-cover h-60 w-full rounded-t-70"
+        style={{
+          backgroundImage: `
+            linear-gradient(180deg, rgba(19, 19, 19, 0) -35.06%, #0F0F0F 100%),
+            url('${bannerImg}')
+          `
+        }}
+      />
+      <div className="bg-black-dark2 p-16 relative rounded-b-70">
+        <div className="absolute top-0 left-16 right-16 flex items-center justify-between">
+          <div className="flex items-center gap-x-3">
+            <div className="relative">
+              <img
+                className="h-20 w-20 rounded-full"
+                src={iconImg}
+                alt={`${title}'s thumbnail`} />
+              <img
+                className="h-10 w-10 rounded-full absolute bottom-0 right-0"
+                src={author.avatar_url}
+                alt={`${author.username}'s avatar`} />
+            </div>
+
+            <div>
+              <h1 className="text-4xl font-bold">{title}</h1>
+              <p className="text-white text-opacity-50 text-2xl">
+                {answer_count} answers submitted
+              </p>
+            </div>
+          </div>
+
+          <div className="p-4 bg-black-lightest rounded-14">
+            <span className="text-4xl font-bold leading-5">Status:</span>
+            <Tag label="Active" theme="success" />
+          </div>
+        </div>
+
+        <Tabs className="mt-16 mb-4">
+          <TabList>
+            <Tab>Info</Tab>
+            <Tab>Responses</Tab>
+          </TabList>
+
+          <TabPanels className="bg-black-lightest px-8 py-5 rounded-b-3xl">
+            <TabPanel>
+              Forum Post
```

Comment:
Pretty sure we already have a field for this, no need to use mocks

Reply:
We don't iirc? Not a BBCode/HTML/Markdown one at least

Reply:
Ok yeah it's probbaly `description` right? But we'll need to add another field for excerpt on the forms listing then

Reply:
Right, @JustRoxy mind adding a text field for short description?